schemas/edges/gtdb/gtdb_child_of_taxon.json

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
-  "required": ["_from", "_to"],
-  "description": "A gtdb_taxon or gtdb_organism is a child of a parent gtdb_taxon in the taxonomy tree.",
+  "required": ["_from", "_to", "source", "child_type"],
```

Comment:
"source" needs to be removed from "required" here

Reply:
Yikes! Completely missed that - thanks for catching it!